Tests/PinEdgesSpec.swift

```diff
@@ -531,6 +561,22 @@ class PinEdgesSpec: QuickSpec {
                 aView.pin.right(20).horizontally()
                 expect(Pin.lastWarningText).to(contain(["horizontally() right coordinate", "won't be applied", "already been set to 20"]))
             }
+            it("should warn") {
+                aView.pin.left(10).horizontally(20)
+                expect(Pin.lastWarningText).to(contain(["horizontally(20.0)", "won't be applied", "already been set to 10"]))
```

Comment:
Once you have updated the method context, thanks to updating the expectation, here it should contain `left`.